timemachine/fe/topology.py

```diff
@@ -253,16 +246,21 @@ def parameterize_nonbonded_pairlist(self, ff_q_params, ff_lj_params):
 
         params = []
         for q, s, e, (sf_q, sf_lj) in zip(q_ij, sig_ij, eps_ij, rescale_mask):
-            params.append((q * sf_q, s, e * sf_lj))
+            params.append(
+                (
+                    q * sf_q,
+                    s,
+                    e * sf_lj,
+                    0.0,  # w offset
+                )
```

Comment:
nit: `s` -> `sig`, `e` -> `eps`?

Reply:
Done in [s -> sig, e -> eps](https://github.com/proteneer/timemachine/pull/893/commits/9c12e4a03859e21b77e91d63aad3f60a323c33f1)